### PR TITLE
zh-trans:/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_…

### PR DESCRIPTION
**Pull request by AdamDang**
…phase_etcd_local.md

zh-trans:/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+
+<!--
+Generates the static Pod manifest file for a local, single-node etcd instance
+-->
+为本地部署且只有一个节点的 etcd 实例生成静态 Pod 清单文件
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md]: 为本地单节点 etcd 实例生成静态 Pod 清单文件

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+<!--
+### Synopsis
+-->
+### 概要
+
+<!--
+Generates the static Pod manifest file for a local, single-node etcd instance and saves it to /etc/kubernetes/manifests/etcd.yaml file. 
+-->
+为本地部署且只有一个节点的 etcd 实例生成静态 Pod 清单文件，并保存到 /etc/kubernetes/manifests/etcd.yaml 文件中。
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md]: 为本地单节点 etcd 实例生成静态 Pod 清单文件

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md (hunk 1 above)
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md]: Done！

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md (hunk 2 above)
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_etcd_local.md]: Done！

**Pull request merged**